Packs/CommonTypes/IndicatorFields/indicatorfield-associations.json

```diff
@@ -26,7 +26,7 @@
     "hidden": false,
     "associatedTypes": null,
     "systemAssociatedTypes": null,
-    "associatedToAll": true,
+    "associatedToAll": false,
```

Comment:
I saw URL and other indicator types using this field in their default mapping. What data returns usefully from this field?

Reply:
Which field? `associations`? don't know, it's an old field